Dockerfile

```diff
@@ -18,27 +18,34 @@ COPY internal/ internal/
 # Copy git to inject the commit during build
 COPY .git .git
 
-# Use FIPS crypto module at build time
-# ARG GOFIPS140=v1.0.0
+# Set the GOEXPERIMENT to enable the strict FIPS runtime check.
+ENV GOEXPERIMENT=strictfipsruntime
 
 # Build
 RUN GIT_COMMIT=$(git rev-list -1 HEAD) && \
     echo " injecting GIT COMMIT: $GIT_COMMIT" && \
-    CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} GOFLAGS=-mod=vendor \
+    CGO_ENABLED=1 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} GOFLAGS=-mod=vendor \
     go build -ldflags "-w -s -X github.com/project-koku/koku-metrics-operator/internal/controller.GitCommit=$GIT_COMMIT" -a -o manager cmd/main.go
 
-FROM registry.redhat.io/ubi9/ubi-micro:latest AS base-env
+# Prepare a ubi micro base with openssl and its dependencies.
+FROM registry.access.redhat.com/ubi9/ubi AS ubi-micro-build
+RUN mkdir -p /mnt/rootfs
+RUN rpm --root /mnt/rootfs --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+RUN dnf install --installroot /mnt/rootfs --releasever 9 --setopt install_weak_deps=false --nodocs -y coreutils-single glibc-minimal-langpack openssl; dnf clean all
+RUN rm -rf /mnt/rootfs/var/cache/*
+
+FROM registry.access.redhat.com/ubi9/ubi-micro AS ubi9-micro
+COPY --from=ubi-micro-build /mnt/rootfs/ /
+CMD /usr/bin/openssl
 
 WORKDIR /
-COPY --from=builder /workspace/manager /usr/bin/costmanagement-metrics-operator
+COPY --from=builder /workspace/manager /manager
```

Comment:
Why did this change? The CSV still references `/us/bin/costmanagement-metrics-operator`:
https://github.com/project-koku/koku-metrics-operator/blob/86d53047d78c4222eb27c41336c1cc9e5bb98102/bundle/manifests/costmanagement-metrics-operator.clusterserviceversion.yaml#L500

Reply:
Nice catch 👀 

Briefly changed the copy destination after running into executable file `/manager not found in $PATH: No such file or directory ` while testing. I've reverted that change.